WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/AddressSelectorCommand.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+import Yosemite
+import SwiftUI
+
+protocol AddressSelectorCommandProtocol {
+    var name: String { get }
+    var code: String { get }
+}
+
+extension Country: AddressSelectorCommandProtocol {}
+extension StateOfACountry: AddressSelectorCommandProtocol {}
```

Comment:
I'm not entirely sure where would be the best place to add these 2 extensions, as are typealias for their Networking models 🤔